src/main/java/com/spotify/docker/client/messages/RegistryAuth.java

```diff
@@ -102,7 +96,8 @@ public final String toString() {
    */
   @SuppressWarnings("unused")
   public static Builder fromDockerConfig() throws IOException {
```

Comment:
can we make this `@Deprecated` if we would like for people to stop using it and to use the RegistryAuthSupplier instead?